pkg/doc/jwt/verifier.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jwt
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"errors"
+	"fmt"
+
+	"github.com/square/go-jose/v3/json"
+	"golang.org/x/crypto/ed25519"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/jose"
+)
+
+const (
+	// signatureEdDSA defines EdDSA alg
+	signatureEdDSA = "EdDSA"
+
+	// signatureRS256 defines RS256 alg
```

Comment:
it would be good to add a NIST P256 test case.

Reply:
there is a TODO at `NewVerifier()` which references https://github.com/hyperledger/aries-framework-go/issues/1266.